dtslint/ts3.8/index.tsx

```diff
@@ -70,3 +79,47 @@ import { SingleDropdown, MultiDropdown } from "../../src/Dropdown";
   value={["foo"]}
   onChange={(_: string[]) => {}}
 />;
+
+<MultiDropdownWithSelectAll selectAllLabel="all" />; // $ExpectError
+
+<MultiDropdownWithSelectAll
+  selectAllLabel="all"
+  options={[]}
+  value={allSelected()}
+  onChange={_ => {}}
+/>;
+
+<MultiDropdownWithSelectAll
+  selectAllLabel="all"
+  options={[]}
+  value={someSelected([])} // could be nice to prevent this error at type level
```

Comment:
What error are you referring to? `someSelected([])` is perfectly legit if the dropdown is clearable (meaning no elements are selected)

Reply:
right, I'll remove this comment